pkg/providers/ovirt/provider.go

```diff
@@ -58,6 +58,18 @@ func NewOvirtProvider(vmiCrName types.NamespacedName, client client.Client, kube
 	return provider
 }
 
+func (o *OvirtProvider) GetVMStatus() (provider.VMStatus, error) {
+	if status, ok := o.vm.Status(); ok {
+		switch status {
+		case ovirtsdk.VMSTATUS_DOWN:
+			return provider.VMStatusDown, nil
+		case ovirtsdk.VMSTATUS_UP:
+			return provider.VMStatusUp, nil
+		}
+	}
+	return "", fmt.Errorf("VM doesn't have a legal status")
```

Comment:
pls include the illegal status in the message

Reply:
Done.